websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,40 @@
+const presence = new Presence({
+		clientId: "658765364439810048"
+	}),
+	date = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		details: "Exploring",
```

Comment:
please only set presenceData details on specific pages or set them at the end of your code if nothing matches

Reply:
okay